src/components/common/modal/modalContainer/ModalContainer.tsx

```diff
@@ -15,13 +15,39 @@
 // along with Superblocks Lab.  If not, see <http://www.gnu.org/licenses/>.
 
 import React, {Component} from 'react';
-import EditModal from '../../../modals/editModal';
-import ShareModal from '../../../modals/shareModal';
-import PreferencesModal from '../../../modals/preferencesModal';
-import LoginModal from '../../../modals/loginModal';
-import ProjectTemplateModal from '../../../modals/projectTemplateModal';
 import style from './style.less';
-import ImportFileModal from '../../../modals/importFileModal';
+import Loadable from 'react-loadable';
+import { Loading } from '../../loadable';
+
```

Comment:
Why making all these modals loadable? They should appear instantaneously. Then they might actually load some data (in which case would be the modal actually showing the loader inside the modal itself)

Reply:
I can answer my own question here. This is to simply delay the modal loading until we actually do need it (so we can reduce the bundle).

Just read this from their own documentation :) 

"Modals, tabs, and many more UI components hide content until the user has done something to reveal it."